Formula/mold.rb

```diff
@@ -82,32 +76,33 @@ def install
     else odie "unexpected compiler"
     end
 
-    system ENV.cc, linker_flag, "test.c"
-    system "./a.out"
-    # Tests use `--ld-path`, which is not supported on old versions of Apple Clang.
-    return if OS.mac? && MacOS.version < :big_sur
+    extra_flags = []
+    extra_flags += %w[--target=x86_64-unknown-linux-gnu -nostdlib] unless OS.linux?
```

Comment:
This `-nostdlib` makes it possible to produce a minimal ELF executable on macOS without requiring `x86_64-elf-gcc` as a test dependency.

Reply:
👍